linera-execution/src/revm.rs

```diff
@@ -455,12 +453,13 @@ where
         _context: OperationContext,
         argument: Vec<u8>,
     ) -> Result<(), ExecutionError> {
+        let argument = serde_json::from_slice::<Vec<u8>>(&argument)?;
```

Comment:
Concretely, this seems to require CLI users to pass a sequence of ASCII bytes encoding a JSON string.